TempDirectory.Test/TempDirectory.Test.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
     <IsPackable>false</IsPackable>
+    <RootNamespace>Messerli.TempDirectory.Test</RootNamespace>

```

Comment:
Enforce for all test classes! `TempDirectoryBuilderTest` is currently in namespace `TempDirectory.Test`

Reply:
Addressed in https://github.com/messerli-informatik-ag/temp-directory/pull/3/commits/9238abf43e161f8c064aabcdd95e220a7993711b